py3status/modules/spotify.py

```diff
@@ -16,6 +16,10 @@
     sanitize_words: which meta data to remove
         *(default ['bonus', 'demo', 'edit', 'explicit', 'extended',
             'feat', 'mono', 'remaster', 'stereo', 'version'])*
+    dbus_client: Used to override which app is used as a client for
```

Comment:
please add this in alphabetical order related to the other options in the docstring

Reply:
done